flow-typed/npm/pull-stream_vx.x.x.js

```diff
@@ -0,0 +1,367 @@
+}
+declare module 'pull-stream/test/drain-abort.js' {
+  declare module.exports: $Exports<'pull-stream/test/drain-abort'>;
+}
+declare module 'pull-stream/test/drain-if.js' {
+  declare module.exports: $Exports<'pull-stream/test/drain-if'>;
+}
+declare module 'pull-stream/test/filter.js' {
+  declare module.exports: $Exports<'pull-stream/test/filter'>;
+}
+declare module 'pull-stream/test/find.js' {
+  declare module.exports: $Exports<'pull-stream/test/find'>;
+}
+declare module 'pull-stream/test/flatten.js' {
+  declare module.exports: $Exports<'pull-stream/test/flatten'>;
+}
+declare module 'pull-stream/test/map.js' {
+  declare module.exports: $Exports<'pull-stream/test/map'>;
+}
+declare module 'pull-stream/test/pull.js' {
+  declare module.exports: $Exports<'pull-stream/test/pull'>;
+}
+declare module 'pull-stream/test/take.js' {
+  declare module.exports: $Exports<'pull-stream/test/take'>;
+}
+declare module 'pull-stream/test/through.js' {
+  declare module.exports: $Exports<'pull-stream/test/through'>;
+}
+declare module 'pull-stream/test/unique.js' {
+  declare module.exports: $Exports<'pull-stream/test/unique'>;
+}
+declare module 'pull-stream/test/values.js' {
+  declare module.exports: $Exports<'pull-stream/test/values'>;
+}
+declare module 'pull-stream/throughs/async-map.js' {
+  declare module.exports: $Exports<'pull-stream/throughs/async-map'>;
+}
+declare module 'pull-stream/throughs/filter-not.js' {
+  declare module.exports: $Exports<'pull-stream/throughs/filter-not'>;
+}
+declare module 'pull-stream/throughs/filter.js' {
+  declare module.exports: $Exports<'pull-stream/throughs/filter'>;
+}
+declare module 'pull-stream/throughs/flatten.js' {
+  declare module.exports: $Exports<'pull-stream/throughs/flatten'>;
+}
+declare module 'pull-stream/throughs/index.js' {
+  declare module.exports: $Exports<'pull-stream/throughs/index'>;
+}
+declare module 'pull-stream/throughs/map.js' {
+  declare module.exports: $Exports<'pull-stream/throughs/map'>;
+}
+declare module 'pull-stream/throughs/non-unique.js' {
+  declare module.exports: $Exports<'pull-stream/throughs/non-unique'>;
+}
+declare module 'pull-stream/throughs/take.js' {
+  declare module.exports: $Exports<'pull-stream/throughs/take'>;
+}
+declare module 'pull-stream/throughs/through.js' {
+  declare module.exports: $Exports<'pull-stream/throughs/through'>;
+}
+declare module 'pull-stream/throughs/unique.js' {
+  declare module.exports: $Exports<'pull-stream/throughs/unique'>;
+}
+declare module 'pull-stream/util/abort-cb.js' {
+  declare module.exports: $Exports<'pull-stream/util/abort-cb'>;
+}
+declare module 'pull-stream/util/keys.js' {
+  declare module.exports: $Exports<'pull-stream/util/keys'>;
+}
+declare module 'pull-stream/util/prop.js' {
+  declare module.exports: $Exports<'pull-stream/util/prop'>;
+}
+declare module 'pull-stream/util/tester.js' {
+  declare module.exports: $Exports<'pull-stream/util/tester'>;
+}
```

Comment:
Well, we can do that right now. pull-stream is part of the entire npm ecosystem, someone else will find it useful and it will save us space and time for when we decide to use it in other places.